### PR TITLE
Update link_microsoft_low_reputation.yml

### PR DESCRIPTION
**Pull request by morriscode**
Excluding Marketing/cold call lingo from MS partners.

### CODE REVIEW

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580, 27556, 27555, 27573, 27550, 26621, 22862, 18649, 2031, 17909, 6798

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580, 27556, 27555, 27573, 27550, 26621, 22862, 18649, 2031, 17909, 6798, 3845, 55271, 111343, 100382, 107025, 104610, 101572, 141968, 110269, 110294

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580, 27556, 27555, 27573, 27550, 26621, 22862, 18649, 2031, 17909, 6798, 3845, 55271, 111343, 100382, 107025, 104610, 101572, 141968, 110269, 110294, 137300, 100695, 117391, 110753, 111567, 110495, 110472, 110474, 110473, 110290

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580, 27556, 27555, 27573, 27550, 26621, 22862, 18649, 2031, 17909, 6798, 3845, 55271, 111343, 100382, 107025, 104610, 101572, 141968, 110269, 110294, 137300, 100695, 117391, 110753, 111567, 110495, 110472, 110474, 110473, 110290, 177876, 141953, 169567, 103298, 121517, 140577, 140595, 143874, 140582, 143884

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580, 27556, 27555, 27573, 27550, 26621, 22862, 18649, 2031, 17909, 6798, 3845, 55271, 111343, 100382, 107025, 104610, 101572, 141968, 110269, 110294, 137300, 100695, 117391, 110753, 111567, 110495, 110472, 110474, 110473, 110290, 177876, 141953, 169567, 103298, 121517, 140577, 140595, 143874, 140582, 143884, 143889, 159328, 159332, 159331, 159330, 159333, 159334, 167276, 159350, 167296

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580, 27556, 27555, 27573, 27550, 26621, 22862, 18649, 2031, 17909, 6798, 3845, 55271, 111343, 100382, 107025, 104610, 101572, 141968, 110269, 110294, 137300, 100695, 117391, 110753, 111567, 110495, 110472, 110474, 110473, 110290, 177876, 141953, 169567, 103298, 121517, 140577, 140595, 143874, 140582, 143884, 143889, 159328, 159332, 159331, 159330, 159333, 159334, 167276, 159350, 167296, 159355, 169365, 169364, 170498, 170500, 170501, 177866, 177864, 177865, 177870

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767, 28681, 28406, 27680, 27657, 27673, 27669, 27577, 27578, 27580, 27556, 27555, 27573, 27550, 26621, 22862, 18649, 2031, 17909, 6798, 3845, 55271, 111343, 100382, 107025, 104610, 101572, 141968, 110269, 110294, 137300, 100695, 117391, 110753, 111567, 110495, 110472, 110474, 110473, 110290, 177876, 141953, 169567, 103298, 121517, 140577, 140595, 143874, 140582, 143884, 143889, 159328, 159332, 159331, 159330, 159333, 159334, 167276, 159350, 167296, 159355, 169365, 169364, 170498, 170500, 170501, 177866, 177864, 177865, 177870, 177868, 177867, 177871, 177869, 177872, 184812, 177874, 184814, 177875, 184815

**Comment by morriscode** [comment]
/mql-mimic-exempt: 177873, 184821, 184885, 184887, 184898, 185418, 185416, 185419, 185417, 185458

**Comment by morriscode** [comment]
/mql-mimic-exempt: 29767,28681,28406,27680,27673,27669,27578,27657,27577,27580,27573,27556,27550,27555,26621,55271,104610,103298,140577,143874,139283,243528,243529,243530,243531,242935,294992,231316

**Comment by morriscode** [comment]
This should help with some FP reduction.

File: detection-rules/link_microsoft_low_reputation.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@ severity: "medium"
 source: |
  type.inbound
  // suspicious link
```
Comment (jkamdjou) [inline detection-rules/link_microsoft_low_reputation.yml:7]: this needs to be moved down

File: detection-rules/link_microsoft_low_reputation.yml [hunk 2]
```diff
@@ -188,18 +196,14 @@ source: |
...
- 
  and (
-    (
-      not profile.by_sender().solicited
-    )
-    or (
-      profile.by_sender().any_messages_malicious_or_spam
-      and not profile.by_sender().any_false_positives
-    )
+   (not profile.by_sender().solicited)
```
Comment (jkamdjou) [inline detection-rules/link_microsoft_low_reputation.yml:200]: parens not necessary here

File: detection-rules/link_microsoft_low_reputation.yml [hunk 3]
```diff
@@ -144,26 +149,29 @@ source: |
...
+                    )
+            )
+     )
+     or (
+       any(ml.nlu_classifier(body.current_thread.text).entities,
+           .name == "urgency"
+       )
+       and not any(ml.nlu_classifier(body.current_thread.text).intents,
+                   .name == "benign" and .confidence == "high"
+       )
```
Comment (jkamdjou) [inline detection-rules/link_microsoft_low_reputation.yml:171]: this one feels a little risky, but  trust your judgement if test rules looks good for a couple days

**Review by jkamdjou — APPROVED** [review]

File: detection-rules/link_microsoft_low_reputation.yml (hunk 3 above)
Comment (morriscode) [reply detection-rules/link_microsoft_low_reputation.yml:166]: This is what's currently in the rule :/  I'll try ripping it out and seeing how it goes!

**Comment by morriscode** [comment]
/mql-mimic-exempt: 52076,159028,141953,121517,140437,140438,140439,140515,169367,161502,243548,250284,253701,249436,249437,245204,245198,273068,268644,268650,297519,297526,297538,297529,297542,297562,314688,266433,314701,314675,154579

**Pull request merged**